File: src/main/java/client/Window.java
package client;

import java.awt.*;

import javax.swing.JFrame;

public class Window extends JFrame{
    public Window (int width, int height, String title, Game game) {
        super(title);
        game.setPreferredSize(new Dimension (width, height));
        game.setMaximumSize(new Dimension (width, height));
        game.setMinimumSize(new Dimension (width, height));
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        setResizable(false);
        setVisible(true);
        requestFocus();
        add(game);
        pack();
        setLocationRelativeTo(null);
    }

    /**
     * TODO :
    private void fullscreen(){
        if (!f11) {
            window.setSize(Toolkit.getDefaultToolkit().getScreenSize());
            window.getGraphicsConfiguration().getDevice().setFullScreenWindow(window);
            window.setVisible(true);
            f11 = true;
        } else {
            window.getGraphicsConfiguration().getDevice().setFullScreenWindow(null);
            window.setSize(new Dimension (WIDTH, HEIGHT));
            window.setVisible(true);
            f11=false;
        }
    }
    **/
}

File: src/main/java/client/Window.kt
package client

import common.Game
import java.awt.Dimension

import javax.swing.JFrame

class Window(width: Int, height: Int, title: String, game: Game) : JFrame(title) {
    init {
        this.preferredSize = Dimension(width, height)
        this.maximumSize = Dimension(width, height)
        this.minimumSize = Dimension(width, height)
        defaultCloseOperation = EXIT_ON_CLOSE
        isResizable = false
        isVisible = true
        requestFocus()
        //add(game)
        pack()
        setLocationRelativeTo(null)
    }
}
